Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -47,6 +47,32 @@ public struct GraphQLHTTPResponseError: Error, LocalizedError {
   }
 }
 
+public struct GraphQLHTTPRequestError: Error, LocalizedError {
```

Comment:
I don't think this needs to conform to `LocalizedError` since it's not providing any localized description of the error.

Reply:
Same reason: I did it because every customised error in project, confirm with both, even though `LocalizedError` already extends from `Error`, and I'm using `errorDescription` declared on this protocol. Of course i can remove it, if you think it would be better in this way. Just didn't want to go against the pattern while opening this PR.

Reply:
Fair enough, i'll talk this one through with @martijnwalraven in meatspace but for this PR, it's fine to leave.